### PR TITLE
Recommended photos table was add

### PR DESCRIPTION
**Pull request by emineayik**
Recommended photos table did added

### CODE REVIEW

File: src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    @NotBlank
+    private String photo;
+
+    @Size(min=1, max = 5)
+    private int rate;
+
+    @Size(min=3, max = 50)
+    private String comment;
+
```
Comment (aykutyurtmen) [inline src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java:33]: please reformat code(ctrl+alt+L)

File: src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package com.daimontech.dsapi.photos.model;
+
+
+import com.daimontech.dsapi.company.model.companyAbout;
```
Comment (aykutyurtmen) [inline src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java]: delete unnecessety library.

File: src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    @Size(min=3, max = 50)
+    private String comment;
+
+
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "user_id", nullable = false)
+    @OnDelete(action = OnDeleteAction.CASCADE)
+    @JsonIgnore
+    private User user;
+
```
Comment (aykutyurtmen) [inline src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java:39]: one blank please.

File: src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java (hunk 2 above)
Comment (emineayik) [reply src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java]: fixed

File: src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java (hunk 3 above)
Comment (emineayik) [reply src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java:39]: fixed

**Review by aykutyurtmen — APPROVED** [review]

File: src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import org.hibernate.annotations.OnDeleteAction;
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+
+@Getter
+@Setter
+@Entity
+@Table(name = "photos")
```
Comment (fiiraz) [inline src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java]: table adı recommended_photos olsa daha iyi

**Review by fiiraz — CHANGES_REQUESTED** [review]
photos table adı recommended_photos olsa daha iyi olur

File: src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java (hunk 4 above)
Comment (emineayik) [reply src/main/java/com/daimontech/dsapi/photos/model/RecommendedPhotos.java]: fixed

**Review by fiiraz — APPROVED** [review]

**Pull request merged**